_targets.R

```diff
@@ -4,6 +4,7 @@ options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c(
   "tidyverse",
   "sf",
+  "rgdal",
```

Comment:
Would recommend avoiding using this package if possible. `sf` was a replacement for the combination of `sp`, `rgdal` and `rgdal` used in the past, and now should cover the functionality of all three of those older packages.

Reply:
Ok, good to know. I couldn't use `sf` alone b/c of it failing to read in the projection information, but now that I'm re-writing the code in R this won't be an issue.